tools/devtool

```diff
@@ -449,6 +449,21 @@ cmd_build() {
     # Cargo uses the debug profile by default. If we're building the release
     # binaries, we need to pass an extra argument to cargo.
     cargo_args=("$@")
+    
+    # Add the default target if we did not get that argument in the build command.
+    add_default_target=true
+    for flag in "${@}"
+    do
+        if [ $flag = "--" ]; then
```

Comment:
Please quote vars (e.g. `[ "$flag" = "value" ]`), when using old-style test constructs, to prevent breakage caused by expansion of multi-word vars.

For instance, if `$flag` was something like `word1 word2`, this would cause a syntax error like `[: too many arguments`.